sigridci/sigridci/markdown_report.py

```diff
@@ -51,6 +51,12 @@ def renderMarkdown(self, analysisId, feedback, options):
         md += "\n----\n\n"
         md += f"- [**View this system in Sigrid**]({sigridLink})\n"
         md += f"- [**View this Sigrid CI feedback in Sigrid**]({landingPage})\n"
+        if options.feedbackURL:
+            md += "\n----\n\n"
+            md += "## Did you find this feedback helpful?\n\n"
```

Comment:
It is clear from the source code here on Github that this is anonymous. Nevertheless, it might be wise to say so, something like: "Did you find this feedback helpful? (will be submitted anonymously)".

Reply:
Well, it's not completely anonymous. It's true we (intentionally) don't know the name of the person, but we do record the system for which feedback is provided.